Geometry/data/QC8GE21/teststand.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0"?>
 <DDDefinition xmlns="http://www.cern.ch/cms/DDL" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cern.ch/cms/DDL ../../../../DetectorDescription/Schema/DDLSchema.xsd">
     <ConstantsSection label="gem21.xml" eval="true">
-        <Constant name="layers"       value="5"/>
+        <Constant name="layers"       value="7"/>
```

Comment:
why do you change this?
is this used?

Reply:
It is not used, I only put it to always keep in mind that QC8 has 7 layers and not 5